### PR TITLE
05 UploadCertificateServiceMockTest, SurveyServiceMockTest, StudyServiceMockTest

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/services/StudyService.java [hunk 1]
```diff
@@ -234,6 +234,8 @@ public final void setBridgeConfig(BridgeConfig bridgeConfig) {
         this.bridgeSupportEmailPlain = bridgeConfig.get(CONFIG_KEY_SUPPORT_EMAIL_PLAIN);
         this.bridgeAdminTeamId = bridgeConfig.get(CONFIG_KEY_TEAM_BRIDGE_ADMIN);
         this.bridgeStaffTeamId = bridgeConfig.get(CONFIG_KEY_TEAM_BRIDGE_STAFF);
+        this.studyWhitelist = Collections.unmodifiableSet(new HashSet<>(
+                BridgeConfigFactory.getConfig().getPropertyAsList(CONFIG_STUDY_WHITELIST)));
```
Comment (DwayneJengSage) [inline src/main/java/org/sagebionetworks/bridge/services/StudyService.java]: I think you intended this to be the passed in bridgeConfig and not the static BridgeConfigFactory.getConfig().

File: src/test/java/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java [hunk 2]
```diff
@@ -52,12 +56,18 @@
...
 import org.sagebionetworks.bridge.models.surveys.SurveyElement;
 import org.sagebionetworks.bridge.models.surveys.SurveyInfoScreen;
+import org.sagebionetworks.bridge.models.surveys.SurveyQuestion;
+import org.sagebionetworks.bridge.models.surveys.SurveyRule;
+import org.sagebionetworks.bridge.models.surveys.SurveyRule.Operator;
 import org.sagebionetworks.bridge.models.surveys.TestSurvey;
+import org.sagebionetworks.bridge.models.surveys.UIHint;
 import org.sagebionetworks.bridge.validators.SurveyPublishValidator;
 
 public class SurveyServiceMockTest {
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java:73]: Missing tests:

* versionSurvey()
* getSurveyAllVersions()

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]

File: src/main/java/org/sagebionetworks/bridge/services/StudyService.java (hunk 1 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/services/StudyService.java]: Yes. Changed.

File: src/test/java/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java (hunk 2 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java:73]: Added

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**